modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -416,6 +418,10 @@ public class TableManager implements IgniteTablesInternal, IgniteComponent {
     @Nullable
     private StreamerReceiverRunner streamerReceiverRunner;
 
+    private CompletableFuture<Void> readyToProcessTableStarts = new CompletableFuture<>();
```

Comment:
please add comment describing the purpose of this future. Also it could be final

Reply:
I have comment here https://github.com/apache/ignite-3/blob/aa414e7839ca6eb8e35cf9009ab45a2a1eaaff7d/modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java#L614 - in the place, where we must be careful about the right ordering of completion.